README.md

```diff
@@ -57,9 +61,13 @@ Commands:
   main  Make a weekly or daily downloads plot
 ```
 
-You can run multiple commands and often give options multiple times. `main`, `freq`, and `all`, support `--key KEY`, thought they choose a nice default. The two "per-week" plots also support a `--daily` flag to change weekly into daily statistics.
+You can run multiple commands and often give options multiple times.
+`main`, `freq`, and `all`, support `--key KEY`, though they choose a nice default.
```

Comment:
@henryiii, I think this is what you meant to say, but correct me otherwise.

Reply:
Yes.